### DISCUSSION TITLE
how to change basepath in Link component?

### DISCUSSION THREAD

**Discussion by ahmadxgani** [molefrog/wouter#323] [Q&A]
in the example:
```jsx
<Router base="/app">
    {/* the link's href attribute will be "/app/users" */}
    <Link href="/users">Users</Link>

    <Route path="/users">The current path is /app/users!</Route>
  </Router>
```

it's possible to change the basepath of `/users`? in my case i want to navigate user from `/dashboard/home` to `/auth/logout` but the actual result is `/dashboard/logout`

i hope i can do something like: `<Link href="/users" base={false}>Users</Link>` to omit or disable chaining base path
i try to change router.base from useRouter hook, but it seems read-only object, am i misunderstand about this section on the [docs](https://github.com/molefrog/wouter#are-relative-routes-and-links-supported)?:
> // we need key to make sure the router will remount when base changed

**Comment by abcd-arl** [comment]
I know this is so late, but you can use `~` to take you to the absolute path.

`<Link href="~/auth/logout">Users</Link>`
